test/test_quantization.py

```diff
@@ -696,6 +698,39 @@ def test_single_liner(self):
             [self.calib_data],
             inplace=False)
         result_eager = model_eager(self.calib_data[0][0])
+        torch._C._jit_pass_quant_fusion(model_script._c._get_module('fc1')._get_method('forward').graph)
+        result_script = model_script._c._get_method('forward')(self.calib_data[0][0])
+        self.assertEqual(result_eager, result_script)
+
+    @_tmp_donotuse_dont_inline_everything
```

Comment:
The default behavior has recently changed to not inline everything by default - please verify that we can remove this decorator.

Reply:
seems to be working